js/screens/Game/GameRound.js

```diff
@@ -73,82 +88,100 @@ export default class GameRound extends React.Component {
         player.isGameOver = isGameOver;
 
-        for (let p of this.players) {
+        this.players.forEach((p) => {
             if (p.isGameOver) {
                 gameOverPlayers++;
             }
-        }
+        });
 
-        if (this.confirmGameOver && (numberOfPlayers - gameOverPlayers) <= 1) {
+        if (this.confirmGameOver && (numberOfPlayers - gameOverPlayers) <= minDeadPlayers) {
             Alert.alert('Game Over', 'What do you want to do?', [
-                { text: 'Restart', onPress: () => self.setState({ roundNumber: this.state.roundNumber + 1 }) },
-                { text: 'Exit', onPress: () => goBack() },
                 {
-                    text: 'Continue', onPress: () => {
+                    text: 'Continue game',
+                    onPress: () => {
                         self.confirmGameOver = false;
-                        Alert.alert('Continue playing', 'Press the back button on your device to exit the game.');
-                    }
-                }
+                        Alert.alert('Continue game', 'Press the back button on your device to exit the game.');
+                    },
+                },
+                { text: 'Exit', onPress: () => goBack() },
+                { text: 'Restart', onPress: () => self.setState({ roundNumber: roundNumber + 1 }) },
             ], { cancelable: true });
         }
     }
 
-    onBackPress = () => {
-        const { navigation } = this.props;
-        if (navigation.index === 0) {
-            return false;
-        }
+    renderSinglePlayer(player, playerProps = {}) {
+        return (
+            <Player
+                {...playerProps}
+                player={player}
+                onGameOver={(isGameOver) => this.playerGameOver(player, isGameOver)}
+            />
+        );
+    }
 
-        Alert.alert('Exit game', 'Are you sure?', [
-            { text: 'Yes', onPress: () => navigation.dispatch(NavigationActions.back()) },
-            { text: 'No' }
-        ]);
+    renderTwoPlayers() {
+        const [player1, player2] = this.players;
+        const { asColumn, rotated180 } = styles;
 
-        return true;
+        return (
+            <View style={asColumn}>
+                <View style={[asColumn, rotated180]}>
+                    {this.renderSinglePlayer(player2)}
+                </View>
+                <View style={[asColumn]}>
+                    {this.renderSinglePlayer(player1)}
+                </View>
+            </View>
+        );
     }
-}
 
-GameRound.propsTypes = {
-    numberOfPlayers: PropTypes.number.isRequired,
-    startingLifeTotal: PropTypes.number.isRequired,
-    onGameEnd: PropTypes.func
-};
+    renderMultiPlayer() {
+        const sides = ['Left', 'Right'];
+        const columns = splitInHalf(this.players);
+        const { column: columnStyle } = styles;
 
-GameRound.defaultProps = {
-    onGameEnd: () => {}
-}
+        return columns.map((players, column) => (
```

Comment:
Ik zou hier een .forEach gebruiken ipv een .map, aangezien een .map een transformed Array returned.
Is maar een super klein dingetje, maar ik zie wel vaak dat .map wordt gebruikt als een .forEach, terwijl er geen transformatie plaats vindt, zou een miniscuul verschil in performance zijn. Just my two cents.